.devcontainer/devcontainer.json

```
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-7.0-jammy",
+	"postCreateCommand": "dotnet restore && dotnet build && dotnet test",
```

We don't need to test I think:

```suggestion
	"postCreateCommand": "dotnet restore && dotnet build",
```